### DISCUSSION TITLE
Visual approaches for ecosystem mapping and sensemaking

### DISCUSSION THREAD

**Discussion by mattosborn** [life-itself/community#1043] [Research]
**UPDATE 2025-08-28**: back-blog this (was supposed to be posted at the time).

This is a small research post in service of our ecosystem sensemaking efforts. It provides some thoughts, reflections and ideas around a variety of relevant visualisation approaches. Some of the referenced works are similar or analogous efforts, and some are unrelated but demonstrate visualisation techniques that might work well when applied here.

## Graph visualisation

![image](https://github.com/life-itself/community/assets/960919/1c5186db-2272-45cd-ae54-a9ccffe54594)

- https://www.illuminatesystems.org/visualisingthefield
- https://www.ama-project.org/explore/?taxonomy=&type=&vis=network
- https://kumu.io/Naryan/the-flourishing-of-all-living-things#flourishing-of-all-living-things/organizations
- https://ich.unesco.org/dive/constellation/?language=en
- https://ich.unesco.org/dive/biome/?language=en

Great for large datasets involving different nodes/entity types and the relationships between them. Can communicate various aspects of a dataset through (a) relationality/links/clustering (b) node size/color/shape/label/etc (c) interaction

When done well it can communicate a bigger picture view, surface patterns, clusters and relationality in complex datasets, and can provide rich interaction for curiosity and exploration.

When done poorly it can leave too much interpretation up to the user, can be visually overwhelming and can be unclear what it is trying to communicate, either through lack of vision, being unrefined, or trying to communicate too much at once. Some of the projects listed above (e.g. [this](https://www.ama-project.org/explore/?taxonomy=&type=&vis=network) and [this](https://kumu.io/Naryan/the-flourishing-of-all-living-things#flourishing-of-all-living-things/organizations)) suffer from some of these issues.
#### Ideas
1. Ecosystem of projects. Clustering by category to give a bigger picture view, and using node styling to communicate individual project attributes. See the [UNESCO project](https://ich.unesco.org/dive/biome/?language=en) for an idea of how this might look.
2. Archetyping. Nodes representing key figures in this space, with clustering based on the "archetype(s)" or "role(s)" that they represent (e.g. educator, sage, connector, visionary, philosopher, peacemaker, mystic, and so on). Node styling to surface attributes of them or their work (AQAL quadrant focus, gender, age, category of work, values).
3. Landscape of actions. Nodes representing actions and practices that have support somewhere in the movement, clustered by category (e.g. inner-work, political change, and so on)

## Radial / layered

![image](https://github.com/life-itself/community/assets/960919/8a12acab-b98f-419e-90e4-3ea1c793317f)

- https://flowingdata.com/2021/07/29/counting-happiness/
- https://cardcaptorsakura.visualcinnamon.com/
- https://ich.unesco.org/dive/domain/?language=en
- https://ich.unesco.org/dive/threat/?language=en

This is a subset of graph visualisations with a layered layout pattern. Nodes are arranged in layers by type, and the relationships are shown between nodes of different layers.

When done well it can densely visualise data in a way that feels ordered, and where using other approaches may feel overwhelming. Can provide a bigger picture perspective. Can be aesthetically pleasing. Works well to visualise the relationships between different types of entities, especially linear or hierarchical ones. Can be more effective at inviting interaction than a traditional graph viz. Can be versatile and adapted to a narrative depending on layout and relationships chosen.

When done poorly it can be overwhelming (too much data) or just fail to be compelling or to communicate a coherent idea.
#### Ideas
1. Ecosystem of projects. Categorising them on various axis, e.g. type, goals, scope, scale, etc
2. Values, goals, actions. Showing the breadth of these things across this movement in its entirety, clustering, categorising and linking them.
3. Key figures. Showing some of the key figures across this movement, along with their ideas, projects or other relevant attributes.

## Flow

![image](https://github.com/life-itself/community/assets/960919/b3718442-d7e0-4534-844b-96a3dcb410ae)

- https://informationisbeautiful.net/visualizations/the-bay-area-memespace-silicon-valleys-most-contagious-ideas/
- https://drive.google.com/file/d/1injm1pLghxgewx5aDoQZr5lh2Qfaf8CW/view?source=post_page-----9393e6ee17d7--------------------------------
- https://twitter.com/exgenesis/status/1562059605148057600/photo/1

Another graph viz sub-type focused more around directionality. Better for modelling and communicating systems and influences. Works better with more distilled ideas, higher level perspectives, refined narratives and curated datasets and layout. More sensemaking work is required in the creation of this, making it more a product of sensemaking than a tool for sensemaking.
## Quadrant

![image](https://github.com/life-itself/community/assets/960919/fbc65c81-574c-4e93-87ba-44c11d7af2b4)

- https://flowingdata.com/2023/09/07/are-we-back-yet/

Simple and effective viz. Wholly dependent on the data having attributes that are interesting when displayed in quadrant format. AQAL quadrant could be an interesting and simple way to display a number of different things here e.g.: (1) key figures (2) projects (3) values (4) goals (5) actions. Node styling, labelling and interaction can be used to communicate other aspects of the dataset.

## Small multiples

![image](https://github.com/life-itself/community/assets/960919/4f562457-39c0-4782-b475-b559324e5b52)

- https://flowingdata.com/2023/09/07/are-we-back-yet/
- https://flowingdata.com/2020/06/18/race-and-origin-in-united-states/

Great for visualising data around a large number of discrete and comparable entities. Can be great to show patterns and outliers or just create a more intuitive impression of a multi-dimensional dataset. Visually compelling and can work well with more interpretive visualisations. The challenges are in design, choosing the right attributes to highlight, and whether or not the underlying data actually has patterns that are interesting to display in this format.
#### Ideas

1. Key figures and their focus. Painting a picture of the kind of work they engage in, such as AQAL quadrants (interior-exterior, individual-collective), or more categorical (philosophical, pragmatic, theoretical, political, environmental, etc)
2. Key figures and archetypes. A comparison of different key figures in this space and the archetypes that they represent (educator, connector, visionary, philosopher, peacemaker, mystic, etc)
3. Projects. A comparison of attributes of existing projects, such as their scope, goals, scale, locality, etc.
4. Project types. Categorical clusters of projects (e.g. environmental projects, community projects, education projects, spiritual projects, etc)

## Nested map 

![image](https://github.com/life-itself/community/assets/960919/f20d799d-595a-40cb-91fa-0e04ec07f215)

- https://kinnu.xyz/

Exploratory way of visualising hierarchical data. This can clearly communicate concepts and knowledge that can be recursively deconstructed. Nested layers could also be other types of visualisation rather than more branch or leaf nodes. This approach may struggle where the hierarchies aren't clean, as is the case with some of the data we could potentially work with.
#### Ideas
1. Knowledge domain -> key thinkers -> their core ideas
2. Knowledge domain -> topic -> sub-topic (e.g. inner growth -> contemplative practice -> meditation)
3. Values -> Goals -> Actions (e.g. environmental sustainability -> regenerative relationship with nature -> permaculture)
4. Tribe/community -> core beliefs -> underlying reasoning
5. Tribe/community -> core beliefs -> utopia/dystopia scenarios

## Skeuomorphic

![image](https://github.com/life-itself/community/assets/960919/5d836249-5417-4da2-a82c-bc9bb8dc3d88)

- https://slatestarcodex.com/blog_images/ramap.html
- https://www.ribbonfarm.com/you-are-here/
- https://slatestarcodex.com/blog_images/ea_imagemap8.html
- https://xkcd.com/802_large/

This is a more playful and interpretive way of visualising a "landscape" of things. If done well this can provide insight, curiosity and reflection, can be engaging, plays nicely off the experience of looking at a real map by emphasising spatial relationships and making good use to metaphor and intuition to communicate less tangible concepts.

The challenge is that this approach is much more an art than a science, it tends to be opinionated and is prone to misrepresenting things, can be too simplistic and is more limited in the density of information it can effectively communicate.
## Scrolly

- https://www.roote.co/wisdom-age
- https://www.reuters.com/graphics/GLOBAL-ENVIRONMENT/INSECT-APOCALYPSE/egpbykdxjvq/index.html

This is a higher level design approach rather than a specific visualisation. The general idea being that you scroll through a web page that incorporates visualisation and media elements, usually in addition to text components. Can provide a linear, "start to finish" experience. This can be an effective way to present a narrative or to communicate ideas in a curated way. When done well can be simple and intuitive and effectively hand-hold the reader through a complex narrative. When done poorly can feel janky, awkward, uninteresting or overwhelming. Works well with a clear goal and a singular and compelling narrative.

## Summary

All of the above have some interesting applications for a sensemaking effort. One big question is *what aspect of this ecosystem are we most drawn to exploring and communicating?*

Other aspects not explored here that vary considerably between different approaches: the datasets and data collection methods required for each, the balance between product of sensemaking vs tool for sensemaking, objective data versus interpretation, and design skills versus technical skills.

My tendency is more towards data heavy and technical approaches (of those explored here that would be graph, radial, small multiples, quadrant) and less towards design and interpretive approaches (flow, skeuomorphic). The former relies more on data collection and technical skills to realise and is more a *tool for sensemaking*, both in the exploration and creation of it and as a finished piece for the end-user. The latter relies more on prior sensemaking work and design skills, and is more a *communication or product of sensemaking*.

**Comment by asimong** [comment]
Just about the only graphing system I have used – because I'm not much of a visual representation person – is concept maps, specifically [Cmap](https://cmap.ihmc.us/) — it's rather old, but it does what it does really well, which is to represent visually the kind of information that is easily represented by linked data / semantic web. If you want to see how I used them, try this for example: https://www.simongrant.org/e/logic-of-competence.html#2011-03-31

**Comment by catherinet1** [comment]
Thanks, @mattosborn, this is a really helpful summary of pros and cons of different options and approaches. 

The distinction between product of sensemaking and tool for sensemaking is useful. My sense is that we would be more inclined right now towards the latter. Unless perhaps, as Rufus suggested, a Skeuomorphic map is explicitly presented as a rough sketch and an invitation for engagement and feedback to contribute to further refinement, as opposed to a finished product. 

For me some questions are:
- What information do we want to (gather) and present?
  - What data do we already have?
  - What further data do we want to collect?
  - What resources do we have to collect further data?

Some of the ideas I like from above are:
- Archetyping. Nodes representing key figures in this space, with clustering based on the "archetype(s)" or "role(s)" that they represent (e.g. educator, sage, connector, visionary, philosopher, peacemaker, mystic, and so on). Node styling to surface attributes of them or their work (AQAL quadrant focus, gender, age, category of work, values).
- Landscape of actions. Nodes representing actions and practices that have support somewhere in the movement, clustered by category (e.g. inner-work, political change, and so on)
- Values, goals, actions. Showing the breadth of these things across this movement in its entirety, clustering, categorising and linking them.
- Key figures. Showing some of the key figures across this movement, along with their ideas, projects or other relevant attributes.
